### PR TITLE
RF | jScript_group > AT_002.001 | Header > After clicking the logo user is redirected to the home page

### PR DESCRIPTION
**Pull request by Bannikova77**
https://trello.com/c/p7zUkgFh/572-rf-jscriptgroup-at002001-header-after-clicking-the-logo-user-is-redirected-to-the-home-page

### CODE REVIEW

File: cypress/e2e/jScript_group.cy.js [hunk 1]
```diff
@@ -15,9 +15,13 @@ describe('Group jScript_group', () => {
         cy.get('#blog-categories [for="weather"] a').should('have.text', this.data.blogPageWeatherFilter);
     });
 
-    it('AT_002.001 | Header > After clicking the logo user is redirected to the home page', function () {
+    it.only('AT_002.001 | Header > After clicking the logo user is redirected to the home page', function () {
```
Comment (Rom-T) [inline cypress/e2e/jScript_group.cy.js:18]: Удалите .only из теста

**Review by LIliia149 — DISMISSED** [review]
Approved

**Review by Rom-T — CHANGES_REQUESTED** [review]
Удалите .only из теста

**Review by LIliia149 — APPROVED** [review]
Approved

**Review by Rom-T — APPROVED** [review]

**Pull request merged**